### PR TITLE
Init logger with log prefix in tracker-server

### PR DESCRIPTION
**Pull request by hanjeonglee**
Init logger with log prefix in tracker-server

### CODE REVIEW

File: tracker-server/logger.js [hunk 1]
```diff
@@ -71,5 +71,20 @@ const winstonLogger = new winston.createLogger({
   exitOnError: false
 });
 
-module.exports = winstonLogger;
+const logger = function(prefix) {
+  const prefixedLogger = {
+    error: function(text) {
+      winstonLogger.error(`[${prefix}] ${text}`)
```
Comment (pengin7384) [inline tracker-server/logger.js:77]: Need ';'

File: tracker-server/logger.js [hunk 2]
```diff
@@ -71,5 +71,20 @@ const winstonLogger = new winston.createLogger({
...
+    },
+    info: function(text) {
+      winstonLogger.info(`[${prefix}] ${text}`)
```
Comment (pengin7384) [inline tracker-server/logger.js:80]: ditto

File: tracker-server/logger.js [hunk 3]
```diff
@@ -71,5 +71,20 @@ const winstonLogger = new winston.createLogger({
...
+    },
+    debug: function(text) {
+      winstonLogger.debug(`[${prefix}] ${text}`)
```
Comment (pengin7384) [inline tracker-server/logger.js:83]: ditto

File: tracker-server/logger.js [hunk 4]
```diff
@@ -71,5 +71,20 @@ const winstonLogger = new winston.createLogger({
...
+    }
+  }
+
+  return prefixedLogger
```
Comment (pengin7384) [inline tracker-server/logger.js:87]: ditto

**Review by pengin7384 — APPROVED** [review]
LGTM with minor comments. Thanks!

File: tracker-server/logger.js (hunk 1 above)
Comment (hanjeonglee) [reply tracker-server/logger.js]: Done. thanks!

**Comment by hanjeonglee** [comment]
PTAL
@pengin7384 @platfowner

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**